.github/workflows/fdroid-publish.yml

```
@@ -0,0 +1,104 @@
+name: Publish to F-Droid
+
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
+
+jobs:
+  publish:
+    if: ${{ secrets.FDROID_SIGNING_KEY != '' && secrets.FDROID_KEY_ALIAS != '' && secrets.FDROID_KEYSTORE_PASS != '' && secrets.FDROID_KEY_PASS != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)